src/main/java/core/basesyntax/machines/Machine.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.machines;
+
+public abstract class Machine {
+    private boolean isWorking;
+    private int machineID;
+
+    protected int getMachineID() {
+        return machineID;
+    }
+
+    protected void setMachineID(int machineID) {
+        if (machineID > 0) {
+            this.machineID = machineID;
+        }
```

Comment:
Let's throw an exception if id is not valid

Reply:
Havent passed Exceptions yet. I'll try.